otx/algorithms/anomaly/configs/classification/padim/template.yaml

```diff
@@ -8,7 +8,7 @@ summary: This model is faster and in many cases more accurate, but it requires a
 application: ~
 
 # Algo backend.
-framework: OTEAnomalyClassification v0.1.0
+framework: OTXAnomalyClassification v0.1.0
```

Comment:
As far as I know, this shouldn't be changed for Geti yet.

Reply:
Thanks for noticing, @samet-akcay could you double check?

Reply:
@goodsong81

Reply:
It seems OK as Geti Platform team has just (yesterday...) merged the framework images into one Docker image (thanks to `pip install .[full]`).

Reply:
![image](https://user-images.githubusercontent.com/4629606/228833638-0c445a5c-38cf-43c5-91fa-ed664e759174.png)